Source/WebKit/Shared/Cocoa/WebCoreArgumentCodersCocoa.serialization.in

```diff
@@ -22,6 +22,44 @@
 
 headers: "ArgumentCodersCocoa.h"
 
+#if USE(CORE_TEXT)
+class WebCore::FontPlatformData {
+    float size();
+    WebCore::FontOrientation orientation();
+    WebCore::FontWidthVariant widthVariant();
+    WebCore::TextRenderingMode textRenderingMode();
+    bool syntheticBold();
+    bool syntheticOblique();
+    WebCore::FontPlatformData::PlatformDataVariant platformSerializationData();
```

Comment:
This should be std::variant<FontPlatformDataSerialization, FontPlatformDataSerializationWithCreationData> or there should be a using statement to put into the metadata what this is.